README.md

```diff
@@ -22,16 +20,16 @@ This client supports the following Google Cloud Platform services:
 * [Container Engine](#container-engine)
 * [Cloud Dataproc](#cloud-dataproc)
 * [Cloud Datastore](#cloud-datastore)
-* [Stackdriver Debugger](#stackdriver-debugger)
-* [Dialogflow API](#dialogflow-api)
+* [Cloud Debugger](#stackdriver-debugger)
```

Comment:
The links need to match the corresponding headings. Since we renamed the section `### Cloud Debugger`, the link needs to be `#cloud-debugger`. Same for the other stackdriver links.

Reply:
Oops, thanks for catching that!